apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -141,38 +158,41 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
   }
 
   public async execute(args?: string[]): Promise<boolean> {
-    this._terminalProvider.verboseEnabled = this.isDebug;
+    // Defensively set the exit code to 1 so if the tool crashes for whatever reason, we'll have a nonzero exit code.
+    process.exitCode = 1;
```

Comment:
Should this get set back to 0 before exiting?

Reply:
It'll get set back to 0 at the end of `onExecute`. See here: https://github.com/microsoft/rushstack/pull/2340/files#diff-6b764ccba8a9718f67ad50c0858e8d8cbc5098442c98a48acdefaea13dbaf131R221